src/App.js

```diff
@@ -11,6 +27,12 @@ const App = () => {
       <main>
         {/* Wave 01: Render one ChatEntry component
         Wave 02: Render ChatLog component */}
```

Comment:
I would generally recommend getting rid of placeholder comments like these once your actual code is in. Just keeps the code a bit cleaner.